isis/src/base/apps/phocube/main.cpp

```diff
@@ -21,6 +21,7 @@ Camera *cam;
 TProjection *proj;
 int nbands;
 bool noCamera;
+bool specialPixels;
```

Comment:
More globals is bad. Use a lambda with capture instead https://github.com/USGS-Astrogeology/ISIS3/wiki/Writing-ISIS3-Tests-Using-Gtest-and-Ctest#process-helper-functions-and-global-variables

Reply:
If you have time it would be good to convert all of the globals to captures, but at least we shouldn't add to them.